website/site/src/components/common/BotDemo.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
```

Comment:
our coding standard defines using single quotes.
I suggest taking temporarilly .prettierc from https://github.com/hubtype/botonic/pull/353 and fixing it in a  commit with no more changes